css/style.css

```diff
@@ -1,9 +1,213 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
-body {
-    font-family: 'Roboto', sans-serif;
+ 
+ body {
+    font-family: 'Roboto',
+    sans-serif;
     -webkit-font-smoothing: antialiased;
+    background-color: white;
+     }
+/*General styles*/
+
+/*Navigation styles*/
+
+#header {
+    background-color: white;
+    border-bottom: 1px solid grey;
+    top: 0;
+    width: 100%;
+}
+
+#logo {
+    width: 30px;
+    padding-left: 200px;
+    float: left;
+}
+
+#nav-bar {
```

Comment:
Please never use IDs in CSS! They are not re-usable like CSS classes, and can cause you trouble with clashing styles.
Always prefer to style a class (`.navigation`) or an HTML element (`p`).
I recommend you swap all your IDs for classes, in your CSS but also in your HTML of course.